### ISSUE TITLE
dModel configuration GUI

### ISSUE THREAD

**Issue by leoglonz** [mhpi/generic_deltaModel#2]
### Summary

A GUI wrapper around a YAML config file for a cleaner and more intuitive user experience, especially for those who do not wish to dive into code.

### Considerations:

- Separate settings into different "folders" in the GUI according to the generic dModel input schema below (for modality parameterization model --> physics model backbone).
- Can the YAML file and GUI write into the same dictionary? (since the GUI part should be removable without consequence for those wishing to directly work with config files/bypass altogether.)

### *Generic input schema*

- General:
    - Mode
    - Random_seed
    - Device
    - GPU_id
    - Main_directory
    - Output_path

- Data & Outputs
    - Name
    - File_paths
    - Area_name
    - Forcing_names
    - Attribute_names
    - Physics_model_forcing_names
    
- Training
    - Start_time
    - End_time
    - Target_feature(s)
    - Batch_size
    - Epochs
    - Save_epoch
    - Run_from_checkpoint

- Testing
    - Start_time
    - End_time
    - Batch_size
    - Test_epoch

- dPL Model
    - Concurrent_model_count (nmul)
    - Nearzero
    - Rho
    - Warm_up
    - Dy_drop

- Physics Model
    - Model(s)
    - Static_parameter_index
    - Dynamic_parameters
         - Parameter_selection_per_model

- Neural Network
    - model_type
    - Dropout
    - Hidden_size
    - Learning_rate

- Loss Function
    - Loss_name
    - Any loss function settings we want exposed
